### PR TITLE
DFPL-1134: make hearingTypeDetails always visible and optional

### PR DESCRIPTION
**Pull request by benouaer**
### JIRA link (if applicable) ###
- https://tools.hmcts.net/jira/browse/DFPL-1134

### Change description ###
- make hearing type details field always visible and optional

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

**Review by KatKovacs1 — APPROVED** [review]

File: ccd-definition/CaseEventToFields/CareSupervision/manageHearings/enterHearingDetails.json [hunk 1]
```diff
@@ -30,11 +30,10 @@
     "CaseEventID": "manageHearings",
     "CaseFieldID": "hearingTypeDetails",
     "PageFieldDisplayOrder": 3,
-    "DisplayContext": "MANDATORY",
+    "DisplayContext": "Optional",
```
Comment (tomxelliott) [inline ccd-definition/CaseEventToFields/CareSupervision/manageHearings/enterHearingDetails.json:33]: I think this needs to be `"DisplayContext": "OPTIONAL"`

**Review by tomxelliott — CHANGES_REQUESTED** [review]
I think the Optional needs to be all caps

File: ccd-definition/CaseEventToFields/CareSupervision/manageHearings/enterHearingDetails.json (hunk 1 above)
Comment (benouaer) [reply ccd-definition/CaseEventToFields/CareSupervision/manageHearings/enterHearingDetails.json:33]: will update

Comment (benouaer) [reply ccd-definition/CaseEventToFields/CareSupervision/manageHearings/enterHearingDetails.json]: done

**Review by tomxelliott — APPROVED** [review]

**Pull request merged**